### ISSUE TITLE
Extracted date shows timezone in a non-RFC2822 format

### ISSUE THREAD

**Issue by PeterWagenaarNW** [mattgwwalker/msg-extractor#69]
**Bug Metadata**
* - [X] Using latest supported version of extract_msg
* Your python version: Python [3.7.2]
* How did you launch extract_msg?
  - [ ] My command line or
  - [X] I used the extract_msg package

**Describe the bug**
The timezone of the date that gets put in the Message.date property isn't RFC2822 formatted.
It is formatted as:
```Mon, 18 Nov 2013 09:26:24 GMT +0100```

The standard describes the format as:
```Mon, 18 Nov 2013 09:26:24 +0100```
So without the GMT.

The problem arises when the date is parsed using the email.utils.parsedate_to_datetime function, which takes timezones into account:

```
In [14]: email.utils.parsedate_to_datetime('Mon, 18 Nov 2013 09:26:24 GMT +0100')
Out[14]: datetime.datetime(2013, 11, 18, 9, 26, 24, tzinfo=datetime.timezone.utc)

In [15]: email.utils.parsedate_to_datetime('Mon, 18 Nov 2013 09:26:24 +0100')
Out[15]: datetime.datetime(2013, 11, 18, 9, 26, 24, tzinfo=datetime.timezone(datetime.timedelta(seconds=3600)))
```

I'm not sure if this is a problem in msg-extractor or in the way it's stored in the msg file. However, if I open the msg file in Outlook on a Windows environment, it shows the Date header as follows:
```Date: Mon, 18 Nov 2013 10:26:24 +0200```

**Comment by TheElementalOfCreation** [comment]
The "gmt" is our fault, but the different time zone might not be. Currently we use a python module to determine the time zone that should be placed onto the msg file. Would you be willing to share the msg file that is generating this discrepancy via email? If so, place it in a zip folder and send it to arceusthe@gmail.com

Keep in mind that those two times are actually the same just in different time zones.

Thank you.

**Comment by PeterWagenaarNW** [comment]
The email in question is actually your own test e-mail inside the repository!

My apologies, I included that in the first draft of the Issue but removed it in the final version. The mail in question is ```msg-extractor/example-msg-files/unicode.msg```

**Comment by TheElementalOfCreation** [comment]
Oh lol. Thanks, well take a look at it.

**Comment by TheElementalOfCreation** [comment]
This should now be fixed in version 0.22.1. Please confirm and close if it has.

**Comment by PeterWagenaarNW** [comment]
It's fixed, thanks!
